CRM/Core/BAO/OptionValue.php

```diff
@@ -518,4 +518,38 @@ public static function getOptionValuesAssocArrayFromName($optionGroupName) {
     return $options;
   }
 
+  /**
+   * Ensure that declared reserved option values exist.
+   */
+  function ensureOptionValues() {
```

Comment:
Ah, nope don't know what /that/ visibility means :/

On 06/15/2015 04:42 PM, Eileen McNaughton wrote:
>
> In CRM/Core/BAO/OptionValue.php 
> <https://github.com/civicrm/civicrm-core/pull/6015#discussion_r32464264>:
>
> > @@ -518,4 +518,38 @@ public static function getOptionValuesAssocArrayFromName($optionGroupName) {
> >      return $options;
> >    }
> >
> > +  /**
> > +   * Ensure that declared reserved option values exist.
> > +   */
> > +  function ensureOptionValues() {
>
> ah - I thought you meant the field in the table
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/civicrm/civicrm-core/pull/6015/files#r32464264>.
>